imageio/plugins/ffmpeg.py

```diff
@@ -91,33 +115,39 @@ def get_exe():
         try:
             with open(os.devnull, "w") as null:
                 sp.check_call([exe, "-version"], stdout=null, stderr=sp.STDOUT)
-                return exe
+                exes.append(exe)
```

Comment:
see https://github.com/imageio/imageio/pull/270/commits/40655463b0c215cd5c4a9d114c2795d3dea0e5e1: I refactored out `resource_package_dir` which is also used in `__main__`.